pages/syllabus.md

```diff
@@ -75,127 +126,142 @@ Class demonstrations, exercises, and homeworks will be done using [jupyter noteb
 
 # Course Modality
 
-This class is offered in person.  I'd say "face to face" but I'll probably wear a KN95 mask most of the time - I, and others in my immediate family, take extra care to avoid even minor colds and the like.  
+This class is offered in person.
+We aim to make class available remotely,
+by broadcasting lectures on zoom and making the recordings available afterwards on Canvas;
+however, at times there are technical issues, so we cannot guarantee this will be available for all lectures.
+Moreover, for most students in-class participation is more engaging and results in better learning outcomes, so we strongly encourage in-person attendance.
+However, we know that things happen in life that might make you unable to come - for instance, please do not come to class if you are sick.
 
-I aim to make class available remotely, by broadcasting lectures on zoom and making the recordings available afterwards on Canvas, however: in the past there've been technical issues making this impossible for more than a few lectures. Moreover, for most students in-class participation is more engaging and results in better learning outcomes, so I strongly encourage in-person attendance. However, I know that things happen in life that might make you unable to come – for instance, please do not come to class if you are sick.
 # Course Policies
```

Comment:
The remaining stuff hasn't actually been changed hardly at all except the Generative AI section; it just looks like it here because of the semantic linebreaks (sorry) and other formatting.